homeassistant/components/squeezebox/media_player.py

```diff
@@ -128,16 +129,19 @@
         ipaddr = socket.gethostbyname(host)
     except (OSError) as error:
         _LOGGER.error("Could not communicate with %s:%d: %s", host, port, error)
-        return False
+        raise PlatformNotReady
```

Comment:
Better would be `raise PlatformNotReady from error`  (includes original error message)